modules/yieldNexusBidAdapter.js

```diff
@@ -0,0 +1,157 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { config } from 'src/config';
+
+const pixKey = 'utrk';
+
+function startsWith(str, search) {
+  return str.substr(0, search.length) === search;
+}
+
+export const spec = {
+  code: 'yieldnexus',
+  aliases: [],
+  supportedMediaTypes: ['banner', 'video'],
```

Comment:
Could import/use the defined mediaTypes from `src/mediaTypes`. Eg `import { BANNER, VIDEO } from 'src/mediaTypes';`. This way you can ensure the values are referenced correctly.